guides/common/modules/con_integrating-a-remote-isc-dhcp-server.adoc

```diff
@@ -0,0 +1,13 @@
+[id="integrating-a-remote-isc-dhcp-server"]
+= Integrating a remote ISC DHCP server
+
+To configure {ProductName} with a remote ISC DHCP server, complete the following procedures:
+
+. xref:configuring-dhcpd-to-use-with-server[]
+. xref:configuring-server-for-use-with-dhcpd[]
```

Comment:
Can we avoid introductions that list subsections like this? Typically just the fact that subsections are included means that users should follow them. If there is a concern that this might not be enough, then perhaps you could use the introduction to explain it, for example "To configure Foreman with a remote ISC DHCP server, configuring `dhcpd` and then configure Foreman Server to use `dhcp`." Just an example of what I have in mind, you might come up with better wording :)

Reply:
Please ignore all intros with only a single sentence or the ones that more or less repeat the title. These are still the old ones. I'll rewrite them in the next update because in some cases, I need additional input from the SMEs to write a good intro (e. g. use case, etc.)

Reply:
For now, I think this is OK but I share Anet's concerns.

Reply:
It was a left over from the old content. I removed this "mini TOC" and rephrased the intro.